libdevcore/OverlayDB.cpp

```diff
@@ -140,7 +140,7 @@ bool OverlayDB::exists(h256 const& _h) const
 
 void OverlayDB::kill(h256 const& _h)
 {
-#if ETH_PARANOIA || 1
+#if 1
```

Comment:
The `#if 1` is useless now.

Reply:
Sure thing, just removed the block

Reply:
Actually, you should not. Leave the `#if 1` block and remove the `#else` one.

Reply:
Sorry, oversight on my behalf - just changed it.